### DISCUSSION TITLE
VIM: The 'v:' scope is not implemented yet :(

### DISCUSSION THREAD

**Discussion by mrpink76** [JetBrains/ideavim#396] [Q&A] [answered]
I'm using Rider and the ideavim plugin.

I'm currently getting the following error briefly showing up in the status bar:
"VIM: The 'v:' scope is not implemented yet :("

What I'm doing is that 
* I have something copied in the default a register, for example: yw
* I put the cursor at the start of a word and do (to paste over the current word): vep

The error occurs when I press the p character.

The strange thing is that this worked previously, but I have reinstalled my computer and rider and now it does not work anymore :P

Anyone that have a clue what might be wrong?

**Comment by lippfi** [comment, marked as answer]
Could you please attach your `.ideavimrc` and `.vimrc` (if you source it)?

**Reply by mrpink76** [reply]
My .vimrc

[vimrc.zip](https://github.com/JetBrains/ideavim/files/7442700/vimrc.zip)

**Reply by mrpink76** [reply]
Now I see that I have a few suspected lines in the bottom of my .vimrc

`xnoremap <expr> p 'pgv"'.v:register.'y`>'`

`xnoremap <expr> P 'Pgv"'.v:register.'y`>'`

I think it has worked with them in my file, but I might be wrong :P

**Reply by lippfi** [reply]
you have following mapping in your vimrc
``` xnoremap <expr> p 'pgv"'.v:register.'y`>'```
in previous versions this mapping was ignored, but since 1.7.0 we support mappings with expressions and now it is executed
but the `v:` scope is not supported for now, so it causes the error

**Reply by mrpink76** [reply]
Ok, thank for the help!
